core/payment/src/dao/activity.rs

```diff
@@ -49,6 +56,50 @@ pub fn set_amount_accepted(
     agreement::increase_amount_accepted(&agreement_id, owner_id, &amount_delta, conn)
 }
 
+pub fn increase_amount_paid(
+    activity_id: &String,
+    owner_id: &NodeId,
+    amount: &BigDecimalField,
+    conn: &ConnType,
+) -> DbResult<()> {
+    assert!(amount > &BigDecimal::zero().into()); // TODO: Remove when payment service is production-ready.
```

Comment:
is there an issue # for this todo?

Reply:
No, there's not. Same with 27 other TODO-s in payments module. We will put that in our JIRA once it is set up.